### DISCUSSION TITLE
Regarding Team Roles

### DISCUSSION THREAD

**Discussion by ramanveerji** [docusealco/docuseal#110] [Q&A]
Is it possible to change the team roles from admin so that they can't access admin panel features?

**Comment by AlexBTurchyn** [comment]
@ramanveerji Roles are not supported in OSS release.

if you want to hide SMTP settings from your OSS users then use ENV variables - other than that i think current feature should cover all use cases for small teams

**Reply by ramanveerji** [reply]
Which envs? Can you explain more.
